src/pages/Invoice/CreateInvoice.js

```diff
@@ -1,7 +1,411 @@
+                                </tr>
+                            </thead>
+                            {fields.items.length > 0 && (
+                                <tbody>
+                                    {fields.items.map((item, idx) => (
+                                        <tr key={idx}>
+                                            <td> {item.name} </td>
+                                            <td>
+                                                <input
+                                                    min="1"
+                                                    type="number"
+                                                    value={item.quantity}
+                                                    onChange={(e) =>
+                                                        updateQuantity(
+                                                            item.id,
+                                                            e.target.value
+                                                        )
+                                                    }
+                                                />
+                                            </td>
+                                            <td>₹{item.price}</td>
+                                            <td>
+                                                ₹
+                                                {item.quantity *
+                                                    Number(item.price)}
+                                            </td>
+                                            <td className="table-action">
+                                                <span
+                                                    onClick={() =>
+                                                        removeElement(item.id)
+                                                    }
+                                                    className="btn-link"
+                                                >
+                                                    <i
+                                                        className="fa fa-trash"
+                                                        aria-hidden="true"
+                                                    ></i>
+                                                </span>
+                                            </td>
+                                        </tr>
+                                    ))}
+                                </tbody>
+                            )}
+                        </table>
+
+                        <div className=" invoice_add-item d-flex align-items-center justify-content-center ">
+                            <span
+                                className="btn-link p-4"
+                                onClick={() => {
+                                    setModalStatus(true);
+                                    setModalType("items");
```

Comment:
combine this into one state 
setModalState({isOpen: true, type: 'items'})

Reply:
i will do this